ansible/playbooks/roles/nginx/files/mime.types

```diff
@@ -0,0 +1,88 @@
+types {
```

Comment:
Is this file modified from the installed version in any way?  I wonder if it'd be simpler to keep the original than to include one in the repo here.